web/src/components/Input/Patient1/InputFiles.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import InputFiles1 from './InputFiles1';
+import InputFiles2 from './InputFiles2';
+import InputFiles3 from './InputFiles3';
+import InputFiles4 from './InputFiles4';
+import './InputFiles.css'
+class InputFiles extends Component {
+ render() {
+    return (
+      <div className="App">
+        <InputFiles1 />
+        <InputFiles2 />
+        <InputFiles3/>
+        <InputFiles4/>
```

Comment:
Here you should also extract a generic component, similar to the comment I left in Configuration.js